ross-nichols-solutions.js

```diff
@@ -0,0 +1,154 @@
+//PART ONE
+//#1
+const calculateCube = (num) => {
+    return num * num * num
+  }
```

Comment:
careful with your indentation here! it looks like from line 5 down, all of your code is indented one level more than it should be. The flip side is that from here down, all your code looks indented consistently, which is great!